src/shared-endpoints.ts

```diff
@@ -0,0 +1,135 @@
+//type Constructor = new (...args: any[]) => {};
+import { GenericAPIResponse } from './util/requestUtils';
+import RequestWrapper from './util/requestWrapper';
+
+export default class SharedEndpoints {
+    protected requestWrapper: RequestWrapper; // XXX Is there a way to say that Base has to provide this?
+    
+    //------------Market Data Endpoints------------>
+    
+    getOrderBook(params: {
+        symbol: string;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/orderBook/L2', params);
+    }
+    
+    /**
+    * @deprecated use getTickers() instead
+    */
+    getLatestInformation(params?: {
+        symbol?: string;
+    }): GenericAPIResponse {
+        return this.getTickers(params);
+    }
+
+    getTickers(params?: {
+        symbol?: string;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/tickers', params);
+    }
+    
+    getSymbols(): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/symbols');
+    }
+    
+    /**
+    * @deprecated use getLiquidations() instead
+    */
+    getPublicLiquidations(params: {
+        symbol: string;
+        from?: number;
+        limit?: number;
+        start_time?: number;
+        end_time?: number;
+    }): GenericAPIResponse {
+        return this.getLiquidations(params);
+    }
+
+    getLiquidations(params: {
+        symbol: string;
+        from?: number;
+        limit?: number;
+        start_time?: number;
+        end_time?: number;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/liq-records', params);
+    }
+    
+    getOpenInterest(params: {
+        symbol: string;
+        period: string;
+        limit?: number;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/open-interest', params);
+    }
+
+    getLatestBigDeal(params: {
+        symbol: string;
+        limit?: number;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/big-deal', params);
+    }
+
+    getLongShortRatio(params: {
+        symbol: string;
+        period: string;
+        limit?: number;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/account-ratio', params);
+    }
+    
+    //------------Market Data Endpoints------------>
+    
+    getApiKeyInfo(): GenericAPIResponse {
+        return this.requestWrapper.get('v2/private/account/api-key');
+    }
+    
+    //------------Wallet Data Endpoints------------>
+    
+    getWalletBalance(params: {
```

Comment:
A bit tedious but can you check that these are ordered like in the docs? Makes it easier to compare when we need to check things: 
https://bybit-exchange.github.io/docs/inverse/#t-wallet

The order for the wallet endpoints is currently:
- getWalletBalance
- getWalletfundRecords
- getWithdrawRecords
- getAssetExchangeRecords